azure-pipelines/ci.yml

```diff
@@ -1,28 +1,58 @@
-# now ci is just about pr validation, 
-# since we tend to make strict configurations for our repo, that is,
-# each pr is up to date before merging into develop and no other way
-# to modify develop except the pr flow.
 trigger: none
 
 pr:
 - main
 - develop
```

Comment:
small side thing, I vote to rename branch `develop` -> `dev` to match our other projects.